package.json

```diff
@@ -13,7 +13,7 @@
 		"url": "https://github.com/dojo/shim.git"
 	},
 	"scripts": {
-		"prepublish": "grunt dist",
+		"prepublish": "grunt peerDepInstall dist",
```

Comment:
Would we want `peerDepInstall` to run on every `dist`?  If so, we should add it to the the `dist` task in grunt-dojo2.  We should also change the dojo2-package-template, if we haven't already, to indicate that the package author should add this to there.

Also, though minor, this does break "only one change per PR" rule, though I break that myself sometimes, but we should create an issue with tasks to retrofit this into all the packages, so we don't forget.